dashboard-api/internal/platform/server/handler/studyconfig/create_test.go

```diff
@@ -0,0 +1,96 @@
+package studyconfig_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/vlab-research/vlab/dashboard-api/internal/platform/storage"
+	"github.com/vlab-research/vlab/dashboard-api/internal/testhelpers"
+	"github.com/vlab-research/vlab/dashboard-api/internal/types"
+)
+
+func TestHandler_StudyConfiguration_Create(t *testing.T) {
+	assert := require.New(t)
+	userId := "auth0|61916c1dab79c900713936de"
+	studyslug := "test-study"
+	testcases := []struct {
+		studyconfig    types.StudyConfig
+		studyslug      string
+		description    string
```

Comment:
should description not fall under the `studyconfig` struct?

Reply:
So this is something called table testing, its just a really fancy way of simplifying your tests into basic structures. The description here is reffering to the test description and not the StudyConfig. Theres a small blog about it [here](https://dev.to/boncheff/table-driven-unit-tests-in-go-407b)

Reply:
gotcha